api/test/robot_tests/web_on_chain/web_on_chain.robot

```diff
@@ -23,8 +23,11 @@ PUT /websiteUploader/prepare
     ...    ${API_URL}/websiteUploader/prepare
     ...    data=${data}
     ...    files=${file}
-    ...    expected_status=${STATUS_OK}
+    ...    expected_status=any
```

Comment:
Shouldn't that be revert to check for `STATUS_OK` ?

Reply:
no, I assert the status code bellow line 30